msbuild/Xamarin.MacDev.Tasks.Core/Tasks/SymbolStripTaskBase.cs

```diff
@@ -1,64 +1,67 @@
 using System;
 using System.IO;
+using System.Collections.Generic;
+
+using Parallel = System.Threading.Tasks.Parallel;
+using ParallelOptions = System.Threading.Tasks.ParallelOptions;
 
 using Microsoft.Build.Framework;
-using Microsoft.Build.Utilities;
+
+#nullable enable
 
 namespace Xamarin.MacDev.Tasks
 {
-	public abstract class SymbolStripTaskBase : XamarinToolTask
+	public abstract class SymbolStripTaskBase : XamarinTask
 	{
 		#region Inputs
 
 		[Required]
-		public string Executable { get; set; }
+		public ITaskItem[] Executable { get; set; } = Array.Empty<ITaskItem> ();
```

Comment:
When building from VS, making this an ITaskItem[] will make XMA copy files from Windows. 
I'm not sure why this wasn't an ITaskItem and how this change will impact the build (do the files we need exist on Windows or those were generated by a previous task on the Mac (in that case the Windows file will be empty)?

We can also avoid copying files to the Mac by ShouldCopyToBuildServer on the SymbolStrip task.

Reply:
@emaf I implemented `ShouldCopyToBuildServer` to get the old behavior.